### PR TITLE
add new layer for LJCO

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

**Comment by lbesnard** [comment]
Not to merged, still need UUID. 
Instrument filter not working

File: workspaces/imos/JNDI_srs_oc_ljco_wws/srs_oc_ljco_wws_daily_wqm_fv02_timeseries_data/filters.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <label>Time</label>
     <visualised>true</visualised>
     <excludedFromDownload>false</excludedFromDownload>
-    <wmsStartDateName>time_start</wmsStartDateName>
-    <wmsEndDateName>time_end</wmsEndDateName>
+    <wmsStartDateName>time_coverage_start</wmsStartDateName>
+    <wmsEndDateName>time_coverage_end</wmsEndDateName>
```
Comment (ggalibert) [inline workspaces/imos/JNDI_srs_oc_ljco_wws/srs_oc_ljco_wws_daily_wqm_fv02_timeseries_data/filters.xml]: Was this change intended in this PR?
